packages/ic-representation-independent-hash/src/representation_independent_hash.rs

```diff
@@ -1,13 +1,14 @@
 use crate::{hash, Sha256Digest};
 use sha2::{Digest, Sha256};
 
-/// Represents a value to be hashed. Only UTF-8 strings and numbers are currently supported.
+/// Represents a value to be hashed. Only UTF-8 strings, bytes and unsigned numbers are currently supported.
 #[derive(Debug, Clone)]
 pub enum Value {
     /// An UTF-8 string to be hashed.
     String(String),
     /// A number to be hashed.
     Number(u64),
+    Bytes(Vec<u8>),
```

Comment:
Clippy will complain about this not having documentation, would you mind adding it?